R/zchunk_LA100.IMAGE_downscale_ctry_yr.R

```diff
@@ -30,61 +29,119 @@ module_aglu_LA100.IMAGE_downscale_ctry_yr_DISABLED <- function(command, ...) {
 
     # Load required inputs
     AGLU_ctry <- get_data(all_data, "aglu/AGLU_ctry")
-    IMAGE_an_Feedfrac_Rimg_C_Sys_Fd_Y <- get_data(all_data, "aglu/IMAGE_an_Feedfrac_Rimg_C_Sys_Fd_Y")
-    IMAGE_an_FeedIO_Rimg_C_Sys_Y <- get_data(all_data, "aglu/IMAGE_an_FeedIO_Rimg_C_Sys_Y")
-    IMAGE_an_Prodmixfrac_Rimg_C_Y <- get_data(all_data, "aglu/IMAGE_an_Prodmixfrac_Rimg_C_Y")
-
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # ===================================================
+    IMAGE_an_Feedfrac_Rimg_C_Sys_Fd_Y <- get_data(all_data, "aglu/IMAGE/IMAGE_an_Feedfrac_Rimg_C_Sys_Fd_Y")
+    IMAGE_an_FeedIO_Rimg_C_Sys_Y <- get_data(all_data, "aglu/IMAGE/IMAGE_an_FeedIO_Rimg_C_Sys_Y")
+    IMAGE_an_Prodmixfrac_Rimg_C_Y <- get_data(all_data, "aglu/IMAGE/IMAGE_an_Prodmixfrac_Rimg_C_Y")
+
+    # Helper function: copy 1970 data to new year 1960
+    extrapolate_1960 <- function(x) {
```

Comment:
Good suggestion, thanks, done. I don't know whether this occurs elsewhere in the code so leaving it in `zchunk_LA100.IMAGE_downscale_ctry_yr.R` for now.